src/main/java/com/epam/ta/reportportal/ws/model/dashboard/CreateDashboardRQ.java

```diff
@@ -33,20 +24,20 @@
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
 /**
  * Domain object for creating dashboards.
  *
  * @author Aliaksei_Makayed
- *
  */
 @JsonInclude(Include.NON_NULL)
 @ApiModel
 public class CreateDashboardRQ extends SharableEntityRQ {
 
-	@NotNull
-	@NotEmpty
-	@Size(min = ValidationConstraints.MIN_NAME_LENGTH,
-	max = ValidationConstraints.MAX_DASHBOARD_NAME_LENGTH)
+	@NotBlank
```

Comment:
https://github.com/hibernate/hibernate-validator/blob/b173247ab36545b90f2bf64a3838f39f4aab2eb1/engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/NotBlankValidator.java#L33

Do we want to allow null values everywhere where NotEmpty and NotNull were?

Reply:
I checked with null and values. It works properly

Reply:
do you mean that null value is allowed in places where @NotNull was before?

Reply:
I mean `@NotBlank` doesnt allow null and empty values. I checked it.

> The @NotNull class’ isValid() method is called after the @NotBlank class’ isValid(), hence forbidding null values.

Reply:
But the documentation of NotBlankValidator says different
https://github.com/hibernate/hibernate-validator/blob/b173247ab36545b90f2bf64a3838f39f4aab2eb1/engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/NotBlankValidator.java#L27

Reply:
But documentation of annotation says that null - not valid. I suppose that one annotation can be handled by many validators in chain.
http://docs.jboss.org/hibernate/stable/validator/api/org/hibernate/validator/constraints/NotBlank.html